app/controllers/application_controller.rb

```diff
@@ -22,21 +22,22 @@ def after_sign_out_path_for(_resource)
   # for production or having setup secrets.
   # If they are not met, render a page with status 500
   def check_requirements
-    fix_secrets, fix_ssl = fixes
-    return unless fix_secrets || fix_ssl
-
-    text = "Please review the following configurations"
-    text += "<ul>"
-    text += "<li>ssl</li>" if fix_ssl
-    text += "<li>secrets</li>" if fix_secrets
-    text += "</ul>"
-    render text: text, status: 500
+    # select any model to check database connection
+    # if database is not configured it will raise an exception
+    User.first
```

Comment:
There are better ways to check that you have a connection to the DB I think. This looks just to hackish...